documentation/asciidoc/topics/ref_anti_affinity.adoc

```diff
@@ -0,0 +1,94 @@
+[id='anti_affinity_configuration-{context}']
+= Anti-Affinity Strategy Configurations
+Review different configurations you can include in your **Infinispan** CR to provide availability for {brandname} clusters.
```

Comment:
I'm not sure what you're trying to say here, but starting with "Review" does not seem correct.

Reply:
reworked this one.